rnote-ui/src/appwindow/appsettings.rs

```diff
@@ -63,6 +63,24 @@ impl RnAppWindow {
             .get_no_changes()
             .build();
 
+        // inertial scrolling
+        self.app_settings()
+            .bind(
+                "inertial-scrolling",
+                &self.settings_panel().general_inertial_scrolling_switch(),
+                "active",
+            )
+            .get_no_changes()
+            .build();
+        self.bind_property(
```

Comment:
It would be better to keep the settings binds as simple as possible, and instead use the `touch-drawing` property to bind the sensitivity. (Also, I'd make the entire row insensitive instead of just the button)

Also: do we need this? It's might not be clear to the user why the setting becomes insensitive and how to make it sensitive again.

Reply:
> But: do we need this? It might not be clear to the user why the setting becomes insensitive and how to make it sensitive again.

I thought about that too, but it has the benefit of showing the user that the option has no effect due to their current configuration. Leaving it sensitive would seem like a bug to me. I guess this comes down to either leaving some users wondering why it doesn't work, or leaving some users wondering why it's not sensitive.

Reply:
Alright, I'd change the subtitle to "Set whether touch scrolling on the canvas is inertial. Always disabled if touch drawing is enabled." then.

Kind of related: Do we want to add punctuation to the subtitles of the other options too? If not, then this one and the DPI setting would be the only ones that have it (because they contain multiple sentences).

Reply:
Sounds good to me :)

Check out the [Gnome HIG writing style](https://developer.gnome.org/hig/guidelines/writing-style.html) (last paragraph), if I understand it correctly it suggests no punctuation if it is a single sentence

Reply:
Well I don't know. I think the guideline got updated recently as well, maybe it used to be different. But I know at least Weblate warns translators if the punctuation differs in the translated string.

And I am also considering the human factor: I don't want to change too much for translators once again, it might be frustrating for them

Reply:
Yeah, fair enough.